src/test/java/org/isf/opd/test/Tests.java

```diff
@@ -88,14 +115,22 @@ public void testOpdGets() throws Exception {
 		_checkOpdIntoDb(code);
 	}
 
+	@Parameterized.Parameters(name = "Test with OPDEXTENDED={0}")
+	public static Collection<Object[]> opdExtended() {
+		return Arrays.asList(new Object[][] {
+				{ false },
+				{ true }
+		});
+	}
+
```

Comment:
This method is such hidden!!

Reply:
what do you mean @emecas ?

Reply:
I meant about moving it before the tests section, since it is more a setup thing, but nothing blocking

Reply:
@dbmalkovsky can you do this?

Reply:
I moved the code; it was a mistake on my part putting it where it was; thanks for noticing it.

Reply:
Thanks a lot @dbmalkovsky !